src/sentry/runner/commands/devserver.py

```diff
@@ -396,7 +396,10 @@ def devserver(
 
     manager = Manager(honcho_printer)
     for name, cmd in daemons:
-        quiet = name not in (settings.DEVSERVER_LOGS_ALLOWLIST or ())
+        quiet = (
+            name not in (settings.DEVSERVER_LOGS_ALLOWLIST or ())
+            or not settings.DEVSERVER_LOGS_ALLOWLIST
```

Comment:
Wouldn't this still be True when the allow list setting is missing? How about something like:

```
and settings.DEVSERVER_LOGS_ALLOWLIST is not None
```

Reply:
you're of course totally right, i should've tested my fix in some way. fixed!